### ISSUE TITLE
Documentation on how to use vue-devtools is missing a crucial step

### ISSUE THREAD

**Issue by nyholku** [vuejs/vue-devtools#1401]
### What problem does this feature solve?
Helps first time users getting started with page:

https://github.com/vuejs/vue-devtools/blob/dev/packages/shell-electron/README.md

While to many people the documentation as it stands is fine and the missing step is obvious there are many even seasoned people like myself that can waste a good 15 minutes to figure this out.

### What does the proposed API look like?
On this page:

https://github.com/vuejs/vue-devtools/blob/dev/packages/shell-electron/README.md

A crucial piece of information or advice is missing.

Every site in the web repeats info from that page but no one seems to mention the important step.

The missing step is that you have to run your vue app as you would run it without dev-tools:

In other words after starting vue tool with 

'vue-devtools' 

execute in shell 

 'npm run serve'

 in the project root directory AND open the page/app in a browser from address

 'http://localhost:8082/'

Do not open try open (in browse)r http://localhost:8098.

Then add the <script src="http://localhost:8098"></script> in to your index.html as follows and RELOAD the page.

<html lang="">
  <head>
    <script src="http://localhost:8098"></script>
